program/program/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿bool sucsessX;
+bool sucsessY;
+string strNumX;
+string strNumY;
+int sum = 0;
+int X;
+int Y;
+
+
+
+do
+{
+    Console.Write("Write X: ");
+    strNumX = Console.ReadLine();
+    sucsessX = int.TryParse(strNumX, out int StrNumX);
```

Comment:
out int StrNumX - code convention violated should be strNumX, but because you have this variable it's better to rename